### PR TITLE
[cpp]Suppress console error when non-weapon in slot <= 3

### LINKED ISSUE #5210: 🐛 getWeaponSkillType error in log
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## OS / platform the server is running (if known)
windows server 2019

## Branch affected by issue 
`base`

## Steps to reproduce
Took awhile to find a repro for this. people were guessing at it in the private servers discord weeks ago, and I finally stumbled on exactly how to see it.

Equip a sword + shield and use vorpal blade. No shield = no error.

This line in `xi.weaponskills.calculateRawWSDmg` will throw an error: 
`local offhandSkill = attacker:getWeaponSkillType(xi.slot.SUB)`

I replaced the get with a zero and used vorpal, no error. put things back, vorpal'd again, error back. Took my shield off and used vorpal: no error. put my shield back on, vorpal again errors. So that's where its coming from for sure.

## Expected behavior
Presumably don't get offhand skill when not a weapon, don't throw error

### PR DESCRIPTION
**Pull request by MowFord**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

`getWeaponSkillType()` lua binding gives a map server error when calling this on a valid slot without a valid weapon. This can be an ammo slot without ammo, ranged slot without weapon, offhand with shield, etc. There's no reason to show an error in those cases. This PR will simply return the same thing that it was returning before, but print nothing in the map server console

Closes #5210 .

## Steps to test these changes

weaponskill with a sword in main hand
- without anything in offhand
- with a shield in offhand
- with a second sword in offhand

See that you get no errors in console and ostensibly weaponskills are still functioning properly.

### CODE REVIEW

**Review by TeoTwawki — APPROVED** [review]
preemptive stamp, but waiting on Ci (and a 2nd approver)

**Review by zach2good — APPROVED** [review]

**Pull request merged**
